src/tests/radsniff/all.mk

```diff
@@ -6,42 +6,57 @@
 #	Test name
 #
 TEST  := test.radsniff
-FILES := $(subst $(DIR)/,,$(wildcard $(DIR)/*.txt $(DIR)/*.gz))
+FILES := $(subst $(DIR)/,,$(wildcard $(DIR)/*.txt))
+
+#
+#	.pcap file to be injested.
+#
+#	It was generated with:
+#	e.g:
+#
+#	TZ=UTC tcpdump -i lo0 -c100 -w radius-auth+acct+coa-100pkts.pcap "port 1812 or port 1813 or port 3799"
+#
+PCAP_IN := $(BUILD_DIR)/tests/radsniff/radius-auth+acct+coa-100pkts.pcap
 
 $(eval $(call TEST_BOOTSTRAP))
 
 #
-#	Run the radsniff commands
+#	Uncompress the input .pcap file
 #
 .PRECIOUS: $(OUTPUT)/%.pcap
 $(OUTPUT)/%.pcap: $(DIR)/%.pcap.gz
 	$(Q)gunzip -c $< > $@
 
-$(OUTPUT)/%.txt: $(DIR)/%.txt $(OUTPUT)/%.pcap $(TESTBINDIR)/radsniff
-	$(eval TARGET   := $(patsubst %.txt,%,$(notdir $@)))
-	$(eval FOUND    := $@)
+#
+#	Run the radsniff commands
+#
+$(OUTPUT)/%.txt: $(DIR)/%.txt | $(TESTBINDIR)/radsniff $(PCAP_IN)
```

Comment:
Why change to  `| $(TESTBINDIR)/radsniff `?  That's an *order* dependency.  i.e. the `radsniff` program has to exist, but the rule isn't re-run if `radsniff` changes.

Just delete the `|` character.

Reply:
yep, done.